apps/prairielearn/src/lib/editors.ts

```diff
@@ -265,16 +316,7 @@ export abstract class Editor {
             job.data.saveSucceeded = true;
           }
         } finally {
-          // Whether or not we error, we'll do a clean and reset.
-          //
-          // If pushing succeeded, the clean will remove any empty directories
-          // that might have been left behind by operations like renames.
-          //
-          // If pushing (or anything before pushing) failed, the reset will get
-          // us back to a known good state.
-          await cleanAndResetRepository(this.course, gitEnv, job);
-
-          // Similarly, whether or not we error, we'll a course sync.
+          // Whether or not we error, we'll a course sync.
```

Comment:
This is probably my fault from ages ago, might as well fix this while we're here.

```suggestion
          // Whether or not we error, we'll sync the course.
```